tests/params.bats

```diff
@@ -306,6 +306,10 @@ setup() {
   assert_output "account-b"
 }
 
-teardown_file() {
-  rm -fv "$DOTENV_FILE" bats.*.yaml
+# teardown_file() {
```

Comment:
Why have you commented removing the generated files and added `cd `$DIR` instead?

Reply:
This looks like an unfortunate mistake. IMHO in the future we should move generated files into temp dir.